README.md

```diff
@@ -56,6 +56,9 @@ this file is unique to each user, and should not be committed to the repository.
 ```TOML
 [bathy]
 BATHY_DATA_DIR = "path to bathymetry data"  # location of bathymetry raster datasets
+
+[ports]
+PORTS_DATA_DIR = "path to port data" # location of port shapefile dataset
```

Comment:
Could I ask why port data is treated separately from other data?

Reply:
I assumed it would be available for people to download on the store or another location like with the Bathymetric data so we don't have to upload it to the repo. Initially I thought it was a larger data folder, but it's just a shapefile. Would you like me to switch to uploading it?

Reply:
I would just add it to the repo for now.

Ideally, `data` would be empty with the expectation that a future user would collate all the data in the exact directory structure needed. This is to avoid any copyright/license issues. Thankfully I think all the data we use is released under Creative Commons so it shouldn't be an issue right now.